pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -1014,7 +1015,28 @@ def on_specialized_clicked(self, button):
         NormalSpoke.on_back_clicked(self, button)
 
     def on_info_bar_clicked(self, *args):
-        if self.errors:
+        if self.disks_errors:
+            label = _("The following errors were encountered when checking your disk "
+                      "selection. You can modify your selection or quit the "
+                      "installer.")
+
+            dialog = DetailedErrorDialog(self.data, buttons=[
+                    C_("GUI|Storage|Error Dialog", "_Quit"),
+                    C_("GUI|Storage|Error Dialog", "_Modify Disk Selection")],
+                label=label)
+            with self.main_window.enlightbox(dialog.window):
+                errors = "\n".join(self.disks_errors)
+                dialog.refresh(errors)
+                rc = dialog.run()
+
+            dialog.window.destroy()
+
+            if rc == 0:
+                # Quit.
+                iutil.ipmi_report(constants.IPMI_ABORTED)
+                sys.exit(0)
```

Comment:
Could you please move the same code (below) to separate method and call it instead.